buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt

```diff
@@ -557,6 +576,7 @@ abstract class PublishingPlugin : Plugin<Project> {
     const val VALIDATE_LIBRARY_GROUPS_TO_PUBLISH_TASK = "validateLibraryGroupsToPublish"
     const val SEMVER_CHECK_TASK = "semverCheckForRelease"
     const val RELEASE_GENEATOR_TASK = "generateReleaseConfig"
+    const val RELEASE_REPORT_GENERATOR_TASK = "generateReleaseReport"
```

Comment:
Please add this to the KDocs for the class where it lists the tasks and functions.